test/clj/cider/nrepl/middleware/debug_integration_test.clj

```diff
@@ -94,11 +94,13 @@
      (nrepl-send {:op :debug-input :input ~(str op) :key (current-key)})))
 
 (def-debug-op :next)
-(def-debug-op :out)
```

Comment:
It's defined just below, as a defmethod, because it takes an optional `force` parameter now.